php-fpm/recipes/cloudwatch.rb

```diff
@@ -0,0 +1,22 @@
+cronscript = '/opt/easybib/bin/phpfpm-cloudwatch.sh'
+
+package 'awscli'
```

Comment:
I would probably use this:
https://github.com/till/easybib-cookbooks/tree/master/awscli

Otherwise:

 1. May conflict with what is installed from the package
 2. The ubuntu package is from the stone age